.github/workflows/slow-checks.yml

```diff
@@ -0,0 +1,44 @@
+name: Slow-Checks
+
+on:
+  workflow_call:
+
+jobs:
+
+  build-matrix:
+    name: Generate Build Matrix
+    uses: ./.github/workflows/matrix-all.yml
+
+  Tests:
+    name: Integration-Tests (Python-${{ matrix.python-version }}, Exasol-${{ matrix.exasol-version}})
+    needs: [ build-matrix ]
+    runs-on: ubuntu-24.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.build-matrix.outputs.matrix) }}
```

Comment:
Given that the integration tests are costly, should this be again reduced to just 3.10? Currently, it'd run for the two specified version 3.10 & 3.11 :grimacing: